FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_Sockets.c

```diff
@@ -1351,17 +1356,16 @@ BaseType_t xReturn;
 			( lOptionName == FREERTOS_SO_SNDBUF ) ? "SND" : "RCV" ) );
 		xReturn = -pdFREERTOS_ERRNO_EINVAL;
 	}
-	else
-	if( ( ( lOptionName == FREERTOS_SO_SNDBUF ) && ( pxSocket->u.xTCP.txStream != NULL ) ) ||
-		( ( lOptionName == FREERTOS_SO_RCVBUF ) && ( pxSocket->u.xTCP.rxStream != NULL ) ) )
+	else if( ( ( lOptionName == FREERTOS_SO_SNDBUF ) && ( pxSocket->u.xTCP.txStream != NULL ) ) ||
```

Comment:
You need `{...}` around the `else` here.  `else if` is not a C language token, so this is an `else` statement with no braces followed by an `if` statement with braces.

Reply:
Done! thanks! :)